### ISSUE TITLE
Is this the true iROSE (2004-2005) source code ?

### ISSUE THREAD

**Issue by SpaceVector** [rminderhoud/irose#4]
I didn't know this was "public" material ? would be glad to contribute

**Comment by rminderhoud** [comment]
Yes, this is the leaked source code from around that time. It's not "pristine" in the sense that it has passed through a few hands who made some modifications along the way but for the most part it's the same. If you get it compiled and setup you'll have the game the same as it was from that time period barring some new minor features (dual skill bars, etc.)

I have this up here just for reference, I don't actively develop it. Many people have their own versions of this code with their own modifications but none of them are public. For the most part this code is difficult to work with and many try to avoid it. If you're interested in rose development maybe you'd be interested in https://github.com/dev-osrose/. The projects there are of a higher quality and more actively developed. There is also a discord: https://discord.gg/JvfyRM7
